lib/dgram.js

```diff
@@ -639,6 +639,25 @@ Socket.prototype.unref = function() {
   return this;
 };
 
+
+Socket.prototype.setRecvBufferSize = function(size) {
+  var err = this._handle.setRecvBufferSize(size);
+
+  if (err) {
+    throw errnoException(err, 'setRecvBufferSize');
+  }
+};
+
+
+Socket.prototype.setSendBufferSize = function(size) {
+  var err = this._handle.setSendBufferSize(size);
```

Comment:
Do you think this would be a better approach? in the wrapper:
```
  if (!args[0]->IsUint32()) {
    return env->ThrowTypeError("size must be an uint32");
  }
```